src/combine_hru_quantiles.slurm

```diff
@@ -14,4 +14,4 @@ mkdir -p /lustre/projects/water/iidd/mhines/logs
 
 module load R/3.5.1-gcc7.1.0
 
-srun Rscript /lustre/projects/water/iidd/mhines/combine_hru_quantiles_david.R
\ No newline at end of file
+srun Rscript /lustre/projects/water/iidd/mhines/combine_hru_quantiles.R
```

Comment:
Thanks, I attempted to use this syntax after getting some feedback from Natalya from Yeti about a script I was struggling to successfully run. When I was able to successfully use this syntax, I did in fact get a much different email from slurm when my job began and when it finished or crashed 😹 . Again I think it would be nice to try to figure this out sometime, but isn't necessary immediately. I did capture these suggestions in an issue in our backlog. https://github.com/usgs-makerspace/makerspace-sandbox/issues/187 I'll likely eliminate them here and commit what works to get this back up to date.

Reply:
Thanks - changing my mind based on the info you shared in that issue and a little [more reading](https://stackoverflow.com/questions/43767866/slurm-srun-vs-sbatch-and-their-parameters). `srun` does indeed seem to be the recommended way to run commands within a submit script. I still don't completely understand why or how it's different from just running `Rscript script.R`, but there must be a good reason, and I'm no longer worried about the syntax.

Reply:
👍 from Natalya's brief sentence, I believe it uses a different set of available hardware bits than you would without using it, therefore more of a 'slave' node sort of dispatch occurs versus with just Rscript, it runs on the node you're on

Reply:
but yeah, i wonder if how we're using this slurm script + using sbatch at the command line kind of becomes moot

Reply:
Yeah...confusing, though I still think you must have it right...I have a call later today with Matt Davis and will report back if I understand more after that.

Reply:
I talked with Matt Davis and Jeff Falgout about `srun` just now. It's a good practice to use - not especially important for my `drake` implementation, but critical when you have n_tasks > 1, because if you don't use srun then the command will always get run on the first allocated task (or node?) whereas if you do use srun then the command will get sent to whatever resources in your job are most available - i.e., only with srun can you make full use of the resources you've requested.